tests/unit-tests/enrolment/test-class-sensei-course-enrolment-manager.php

```diff
@@ -200,4 +250,40 @@ private function create_course_enrolment_mock( $course ) {
 		return $mock;
 	}
 
+	/**
+	 * Assert that an enrolment check was deferred.
+	 *
+	 * @param int $user_id   User ID.
+	 * @param int $course_id Course post ID.
+	 */
+	private function assertEnrolmentCheckDeferred( $user_id, $course_id, $message = null ) {
+		$property = new ReflectionProperty( Sensei_Course_Enrolment_Manager::class, 'deferred_enrolment_checks' );
+		$property->setAccessible( true );
+		$deferred = $property->getValue( Sensei_Course_Enrolment_Manager::instance() );
+
+		$this->assertTrue( isset( $deferred[ $user_id ][ $course_id ] ), $message );
+	}
+
+	/**
+	 * Assert that an enrolment check was NOT deferred.
+	 *
+	 * @param int $user_id   User ID.
+	 * @param int $course_id Course post ID.
+	 */
+	private function assertEnrolmentChecNotDeferred( $user_id, $course_id, $message = null ) {
```

Comment:
Missing the letter "k" in assertEnrolmentChec**k**NotDeferred

Reply:
Nice checing 😄  b353a7e372f7d3a6713cae7657ba162f4dedb0f6